ui/packages/catalog-ui-search/src/main/webapp/component/search-form-list/search-form-list.view.js

```diff
@@ -20,24 +20,51 @@ const Router = require('../router/router.js')
 const user = require('../singletons/user-instance')
 const SearchForm = require('../search-form/search-form')
 import React from 'react'
+import { lighten, readableColor, transparentize } from 'polished'
 import styled from '../../react-component/styles/styled-components'
 
+const ListContainer = styled.div`
```

Comment:
Could the react parts of this file be moved to its own file?

Reply:
I'm tempted to move the React components into their own file but I think that just might add more indirection. This file will (hopefully) only contain these components anyway once the remaining Marionette code is migrated.